pkgs/development/libraries/ncurses/default.nix

```diff
@@ -173,7 +183,7 @@ stdenv.mkDerivation rec {
   };
 
   passthru = {
-    ldflags = "-lncurses";
+    ldflags = "-lncurses -ltinfo";
```

Comment:
Totally unsure whether this is appropriate, hopefully someone with more experience can chime in.

Reply:
Rebased & removed this.